include/kernel-version.mk

```diff
@@ -4,11 +4,11 @@ LINUX_RELEASE?=1
 
 LINUX_VERSION-3.18 = .43
 LINUX_VERSION-4.4 = .56
-LINUX_VERSION-4.9 = .17
+LINUX_VERSION-4.9 = .18
 
 LINUX_KERNEL_HASH-3.18.43 = 1236e8123a6ce537d5029232560966feed054ae31776fe8481dd7d18cdd5492c
 LINUX_KERNEL_HASH-4.4.56 = 55181e24869e4b550ef756eecd1ab4f6774228c457f4830e9e779c6db4c2a2b6
-LINUX_KERNEL_HASH-4.9.17 = 9095c16550ebae9cbe0979a706116d95fbd8d270c1068857c4a8788de0f9ac7f
+LINUX_KERNEL_HASH-4.9.18 = a10eea12a85667636ea2503aebcdae3005c821333c489fc54e2a4e9fc44261d1
```

Comment:
This file still refers to 4.9.18 although you have already changed the commit title for .19